rootfs-builder/rootfs.sh

```diff
@@ -246,6 +246,8 @@ OK "Pull Agent source code"
 
 info "Build agent"
 pushd "${GOPATH_LOCAL}/src/${GO_AGENT_PKG}"
+[ -n "${AGENT_VERSION}" ] && git checkout "${AGENT_VERSION}" || true
+OK "git checkout successful"
```

Comment:
nit: move OK up to after `git checkout`?

Reply:
Do you want to replace `true` with `OK` func, IMO, it's inconsistent with the other lines, such as line 244 ~245. Or did i misunderstand your comment?☺

Reply:
line 244~245 is different because `go get` is always executed, where as `git checkout` here is optional but we will always print "git checkout successful". Anyway, it's just minor nits. I'm fine if you want to merge it as is.

Reply:
you are right😁. We just optionally `git checkout`.  I will put `OK `after `git chekcout` asap.